### ISSUE TITLE
DetailsList headers are read multiple times by NVDA

### ISSUE THREAD

**Issue by brandonthomas** [microsoft/fluentui#14326]
### Environment Information

- **Package version(s)**: 7.124.0 but I think this has been around for a while
- **Browser and OS versions**: Chrome, Edge Chromium, Firefox with NVDA

### Describe the issue:
When navigating a DetailsList the headers are read out in an odd way. They repeat the header name 3 times, and mention sorting twice. Overall it is somewhat confusing and verbose.

Might be related to #13779

### Please provide a reproduction of the issue in a codepen:

https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist

#### Actual behavior:
Name header in the example above reads out "name column header sorted ascending row 1, name column 2, name button sorted a-z row 1" in NVDA. It's just a lot of info that isn't actually helpful.

#### Expected behavior:
I would expect to get each piece of info only once. The header name is "name", and how it's sorted, and that it's interactive.

**Comment by khmakoto** [comment]
Hi @brandonthomas on my end I do hear the "name" portion being repeated twice, but I cannot get it to repeat the sort portion twice. Furthermore, I checked and the "name" portion repeats twice in other screen readers such as Narrator as well, so I'm tagging this as a bug that needs to be fixed and will get to it as soon as possible.

**Comment by khmakoto** [comment]
Actually, on further investigation, I think this is intended behavior because we are using nested components with two different roles: "columnheader" and "button" which are read at the same time when focus stops on the header. That's why the "name" is pronounced twice because of how screen readers work. Given that we want to maintain both roles so that all the appropriate information is read by screen readers I think it is not possible to stop the duplication of information. However, I think it is a compromise that's worth making to have more or duplicate information vs losing any of it.

**Comment by brandonthomas** [comment]
@khmakoto can we follow up with the accessibility team on this? The trusted testers filed a bug on this one so I am not sure that it's right and usually AT users don't like this much verbosity.

**Comment by khmakoto** [comment]
Sure, @smhigley could you help us here?

**Comment by brandonthomas** [comment]
bump!

**Comment by alixiashu** [comment]
Hi @brandonthomas, has this issue been fixed in some version or on the plan? I could still meet it when testing on the example
https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/basic

**Comment by brandonthomas** [comment]
It's not owned by me to be clear. I just filed this particular issue. @khmakoto any update here on the Fluent team side?

**Comment by khmakoto** [comment]
Pinging @smhigley one more time 😄

**Comment by smhigley** [comment]
@brandonthomas I can't reproduce this issue, NVDA behaves as expected for me. I get the following output on Firefox (84.0.2), and Chrome (88), with NVDA v.2020.1:

- in Firefox, when navigating on the headers, it reads the column name once. Moving through body cells within that column it also only reads the header once. 
- in Chrome, it reads "column name, button column name" on the header (essentially reading first the header cell, then its content). It only reads the column name once when moving through body cells.

Both results are expected, and match up with how other tables behave using NVDA + those browsers. I can't reproduce the issue with NVDA reading the header name three times. I'd ask your tester if they're using table commands to test the tables -- sometimes people who don't use a screen reader themselves make that mistake.

If they are using table commands, could you provide a video, or more details on NVDA version, browser versions, and the exact interactions the tester is using? Or maybe a log of the actual speech output would help.

**Comment by khmakoto** [comment]
We've followed offline with this issue and it's no longer relevant, so we'll be closing this as `Resolution: External`.
